### PR TITLE
v-get helper

### PR DESCRIPTION
**Pull request by offirgolan**
- [x] Write `v-get` to `get` AST Transformation
- [x] Get this working with `ember-get-helper` for Ember 1.10 to 1.13.9
- [x] Test cases
- [x] Test out blueprint with version checker
- [x] Update README

Previous discussion can be found at #12 

If you want to play around with the AST Transformation you can just clone this [JSBin](http://jsbin.com/xuyaye/20/edit?html,js,output)

### CODE REVIEW

**Comment by rwjblue** [comment]
Testing with the addon for 1.10 - 1.13 support should only require adding it as a devDep. The ember-get-helper addon already [noops itself when running with 2.0.0](https://github.com/jmurphyau/ember-get-helper/blob/master/index.js#L15) (but it does print a warning which will be annoying).

**Comment by offirgolan** [comment]
@rwjblue I'm actually having some issues getting this to work with `ember-get-helper` in Ember 1.13. Seems like it's due to a deprecation to RegisterHelper.

**Comment by rwjblue** [comment]
> As for the inclusion in package.json, I dont think we should include it if it's not necessary.

I am saying that we include it in our `devDependencies` when testing the library itself under 1.x.  Not that we should add it to `dependencies` (which would have all downstream consumers get a warning)

File: package.json [hunk 1]
```diff
@@ -41,10 +41,12 @@
     "ember-cli-moment-shim": "0.6.1",
     "ember-cli-qunit": "1.0.1",
     "ember-cli-uglify": "^1.0.1",
+    "ember-cli-version-checker": "^1.1.4",
```
Comment (rwjblue) [inline package.json]: For this to be usable in the blueprint, it needs to be included in `dependencies` not `devDependencies`.

**Comment by offirgolan** [comment]
>  I am saying that we include it in our devDependencies when testing the library itself under 1.x. Not that we should add it to dependencies (which would have all downstream consumers get a warning)

The previous commit was using Ember 2.0.1, that's why it wasn't in the dependencies. I reverted back to 1.13.8 and included the ember-get-helper in the devDep.

On a side note, would it be possible to test the AST Transformations via integration testing like a component? If not, ill write up some unit tests.

File: package.json (hunk 1 above)
Comment (offirgolan) [inline package.json]: :+1:

**Comment by rwjblue** [comment]
Yes, you can test any template snippet via `moduleForComponent` (it doesn't need to be related to a component in any way).

Demo: http://jsbin.com/daloriq/edit?js,output

**Comment by offirgolan** [comment]
@rwjblue @stefanpenner @blimmer this is done from my end. Feel free to review when you get some time so we can merge this baby.

File: tests/integration/helpers/v-get-test.js [hunk 2]
```diff
@@ -0,0 +1,88 @@
+import Ember from 'ember';
+import { moduleForComponent, test } from 'ember-qunit';
+import hbs from 'htmlbars-inline-precompile';
+import '../../helpers/ensure-get-registered';
+
+moduleForComponent('helper:v-get', 'Integration | Helper | v-get', {
+  integration: true,
+  beforeEach() {
+    var model = Ember.Object.create({
```
Comment (stefanpenner) [inline tests/integration/helpers/v-get-test.js]: i suspect this will work without an ember-object.

**Comment by stefanpenner** [comment]
SGTM. would like @rwjblue's +1 on the transform code though.

**Comment by rwjblue** [comment]
Tests seem to cover all of the various cases in the transform and are passing on all supported versions.

LGTM

**Comment by blimmer** [comment]
Sorry to be late to the party on looking at this. Looks great - the AST stuff is cool and makes it way nicer to work with these long strings.

**Comment by jasonmit** [comment]
Will this work at as a runtime plugin as well?  I worked around this in the past:  https://github.com/emberjs/legacy-bind-attr/blob/master/index.js#L12

**Comment by rwjblue** [comment]
@jasonmit - No, it does not work as a runtime plugin as it stands today. We could absolutely do a similar thing to legacy-bind-attr, but I'd prefer for that to be behind a flag if we do (no need to bloat every users `vendor.js` when 95% of them won't use it).  In legacy-bind-attr we needed it to be enabled by default for backwards compat reasons, we don't really have that issue here...

**Pull request merged**